config/routes.rb

```diff
@@ -27,7 +27,7 @@
 
   root to: 'pages#show'
 
-  resources :signs, only: :show, module: SignModel.adapter.to_sym do
+  resources :signs, only: :show, module: :signbank do
```

Comment:
I'd like to pull these controllers up to the top-level now we don't have multiple adapters